### PR TITLE
[1070] Enable history tracking for data entry/edit

### PR DESCRIPTION
**Pull request by stephenhillier**
* change history tracking is enabled for companies and drillers (including registrations/classifications associated with a given driller)
* endpoints are `/gwells/api/v1/drillers/<id>/history/` and `/gwells/api/v1/organizations/<id>/history/`
* simple frontend component that displays the list of changes for either companies or people

### CODE REVIEW

File: api-tests/registries_api_tests.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"info": {
-		"_postman_id": "78c45691-2533-486f-af0c-78cac2d05f53",
-		"name": "GWELLS API remote",
+		"_postman_id": "4e69cb6c-770f-4ff3-a298-f37095fcee23",
+		"name": "GWELLS API remote copy",
```
Comment (Sybrand) [inline api-tests/registries_api_tests.json]: Could we drop "copy" from the name?

**Comment by stephenhillier** [comment]
[EPIC#Registry | Start tracking change history for records |2| !! [gherkintodo]](https://trello.com/c/IhCr7JkB/1070-epicregistry-start-tracking-change-history-for-records-2-gherkintodo)

**Comment by stephenhillier** [comment]
[EPIC#Registry | Implement view Business History (Front end component) |2| !! [gherkintodo]](https://trello.com/c/BYMFzzIj/1067-epicregistry-implement-view-business-history-front-end-component-2-gherkintodo)

**Comment by stephenhillier** [comment]
[EPIC#Registry | Implement Business History API (Companies) |3| !! [gherkintodo]](https://trello.com/c/Q2Tqqve6/1009-epicregistry-implement-business-history-api-companies-3-gherkintodo)

File: app/registries/urls.py [hunk 2]
```diff
@@ -52,11 +53,13 @@
     url(r'^api/v1/organizations/names/$',
         never_cache(views.OrganizationNameListView.as_view()),
         name='organization-names'),
+    url(r'^api/v1/organizations/(?P<org_guid>[-\w]+)/history/$',
+        views.OrganizationHistory.as_view(), name='organization-history'),
```
Comment (Sybrand) [inline app/registries/urls.py]: This should probably also be never_cache?

File: app/registries/urls.py [hunk 3]
```diff
@@ -68,6 +71,8 @@
     # Person resource endpoints (drillers, well installers, and other instances of Person model)
     url(r'api/v1/drillers/options/',
         views.PersonOptionsView.as_view(), name='person-options'),
+    url(r'^api/v1/drillers/(?P<person_guid>[-\w]+)/history/$',
+        views.PersonHistory.as_view(), name='person-history'),
```
Comment (Sybrand) [inline app/registries/urls.py]: never_cache?

File: app/registries/views.py [hunk 4]
```diff
@@ -647,7 +664,90 @@ class OrganizationNoteDetailView(AuditUpdateMixin, RetrieveUpdateDestroyAPIView)
...
+    """
+    get: returns a history of changes to an Organization model record
+    """
+
+    permission_classes = (GwellsPermissions,)
+    queryset = Organization.objects.all()
+    swagger_schema = None
+
+    def get(self, request, org_guid):
+        org_guid = self.kwargs['org_guid']
```
Comment (Sybrand) [inline app/registries/views.py]: Always ignoring/replacing function param org_guid ?

File: app/registries/views.py [hunk 5]
```diff
@@ -647,7 +664,90 @@ class OrganizationNoteDetailView(AuditUpdateMixin, RetrieveUpdateDestroyAPIView)
...
+    queryset = Person.objects.all()
+    swagger_schema = None
+
+    def get(self, request, person_guid):
+        """
+        Retrieves version history for the specified Person record and creates a list of diffs
+        for each revision.
+        """
+
+        person_guid = self.kwargs['person_guid']
```
Comment (Sybrand) [inline app/registries/views.py]: Same as above, passing person_guid, but always replacing with self.kwargs value

**Pull request merged**